doc/next/_sources/reference/get-campaigns.rst.txt

```diff
@@ -0,0 +1,46 @@
+.. index:: GET /campaigns
+.. _get_campaigns:
+
+GET /campaigns
+==================
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.campaign.list-v2+json``
+
+Returns a list of campaigns which belong to the user. At the moment there is a restriction of *at most* one campaign per seller. This restriction will be lifted-off in later stages.
```

Comment:
belong to the Seller. Why do we use Seller and user in one sentence? what is the standard across the docs?

Reply:
I wondered the same while writing. We don't have a standard glossary of the product or anything that I know

Reply:
I would default to user, more common lingo likely with the partner their API's

Reply:
who can enforce standardization of terminology, so we can apply it to API docs as well?

Reply:
I wish it was that simple :( but we even have it in some fields in the API. 
From now on, `user` it is, at least in what we surface to the outside world.